src/test/kotlin/uk/gov/justice/digital/hmpps/personrecord/test/responses/AllProbationCasesResponses.kt

```diff
@@ -0,0 +1,121 @@
+package uk.gov.justice.digital.hmpps.personrecord.test.responses
+
+fun allProbationCasesSingleResponse(firstCrn: String, firstPrefix: String) = """
+  {
+    "content": [
+        {
+            "identifiers": {
+                "deliusId": 2500000501,
+                "crn": "$firstCrn"
+            },
+            "name": {
+                "forename": "${firstPrefix}FirstName",
+                "surname": "${firstPrefix}LastName"
+            },
+            "dateOfBirth": "1980-08-29",
+            "gender": {
+                "code": "M",
+                "description": "Male"
+            },
+            "aliases": [],
+            "addresses": []
+        }],
+    "pageable": {
+        "pageNumber": 6004,
+        "pageSize": 100,
+        "sort": {
+            "unsorted": false,
+            "sorted": true,
+            "empty": false
+        },
+        "offset": 600400,
+        "paged": true,
+        "unpaged": false
+    },
+    "totalElements": 7,
+    "totalPages": 4,
+    "last": false,
+    "numberOfElements": 2,
+    "first": true,
+    "size": 2,
+    "number": 1,
+    "sort": {
+        "unsorted": false,
+        "sorted": true,
+        "empty": false
+ },
+    "empty": false}
+""".trimIndent()
```

Comment:
Minor but could have a new line inbetween functions

Reply:
Agreed - I feel like ktlint should be picking up this kind of thing for us